helmfile/helmfile.yaml

```diff
@@ -253,6 +251,8 @@ releases:
     values:
     - ./overrides/system/fluentbit-agent.yaml.gotmpl
 
+{{ if or (eq .Environment.Name "dev") (eq .Environment.Name "staging") }}
```

Comment:
I don't know how/why this got put back - cert-manager is in prod w/ helm... Can you remove this if statement?